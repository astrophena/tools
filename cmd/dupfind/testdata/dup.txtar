-- a.txt --
hello
-- b.txt --
hello
-- c.txt --
world
-- d.txt --
world
