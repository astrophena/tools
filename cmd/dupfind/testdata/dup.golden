[{cur:b.txt prev:a.txt} {cur:d.txt prev:c.txt}]